frontend/src/pages/LandlordPage.tsx

```diff
@@ -375,15 +365,15 @@ const LandlordPage = ({ user, setUser }: Props): ReactElement => {
                   style={{ width: '107px', height: '43px' }}
                 />
               </IconButton>
-              <Button
+              {/* <Button
```

Comment:
This doesn't impact the code too much and you probably just overlooked it, but maybe just delete the button completely instead of leaving it as a comment to simplify the code.